packages/opentelemetry-tracing/src/MultiSpanProcessor.ts

```diff
@@ -25,7 +25,9 @@ export class MultiSpanProcessor implements SpanProcessor {
   constructor(private readonly _spanProcessors: SpanProcessor[]) {}
 
   forceFlush(): void {
-    // do nothing as all spans are being exported without waiting
+    for (const spanProcessor of this._spanProcessors) {
+      spanProcessor.forceFlush();
```

Comment:
What happens if one of these throws?

Reply:
Just like other MultiSpanProcessor methods, it just throws. Basically, MultiSpanProcessor doesn't have anything to do with these errors. So if there are no other ideas, I'm not suggesting swallowing errors and let 'em crash.

Reply:
That's fine with me just wanted to make sure we thought about it.